experiments/golden-results/UKNI-Information-Barrier-summary.txt

```diff
@@ -107,3 +112,8 @@ Occurs: 1 times
 Redacted compiler error message:
 unmatched actual "REDACTED" in call
 Raw compiler error message:
+--
+Occurs: 1 times
++===========================GNAT BUG DETECTED==============================+
+| GNU Ada (ada2goto) Assert_Failure failed precondition from range_check.ads:29|
```

Comment:
It seems that the type with the `size` clause is used in a type conversion later in the program. That is currently unsupported. So it probably only emerged now because we started handling the 'size' clause. We might want to fix the range-check for type conversion in a different PR.

Reply:
Yes; definitely do and also in a separate PR.